pkg/server/manager/stack/util.go

```diff
@@ -274,8 +274,8 @@ func (m *StackManager) BuildStackFilter(ctx context.Context, query *url.Values)
 			return nil, err
 		}
 		filter.ProjectID = projectEntity.ID
-		if envParam != "" {
-			filter.Path = fmt.Sprintf("%s/%s", projectEntity.Path, envParam)
+		if pathParam != "" {
```

Comment:
no need to prefix with project path
`filter.Path = pathParam` should be sufficient